web/src/main/java/no/nav/foreldrepenger/web/app/tjenester/fordeling/OpprettSakTjeneste.java

```diff
@@ -37,50 +31,50 @@ public OpprettSakTjeneste(FagsakTjeneste fagsakTjeneste, NavBrukerTjeneste bruke
         this.brukerTjeneste = brukerTjeneste;
     }
 
-    public Fagsak opprettSakVL(AktørId bruker, FagsakYtelseType ytelseType) {
+    Saksnummer opprettSak(FagsakYtelseType ytelseType, AktørId aktørId, JournalpostId journalpostId) {
```

Comment:
Har vi fortsatt behov for opprettSak uten journalpost her? Det skjer ved informasjonssaker men der brukes ikke denne - men opprettFagsak(FagsakYtelseType ytelseType, NavBruker bruker)